t/Equaldex.t

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Test::More;
+use DDG::Test::Spice;
+
+spice is_cached => 1;
+
+ddg_spice_test(
+    [qw( DDG::Spice::Equaldex)],
+
+    'lgbt china' => test_spice(
+        '/js/spice/equaldex/china',
+        call_type => 'include',
+        caller => 'DDG::Spice::Equaldex'
+    ),
+
+    'lesbian rights cyprus' => test_spice(
+        '/js/spice/equaldex/cyprus',
+        call_type => 'include',
+        caller => 'DDG::Spice::Equaldex'
+    ),
+    
+    'gay rights australia' => test_spice(
+        '/js/spice/equaldex/australia',
+        call_type => 'include',
+        caller => 'DDG::Spice::Equaldex'
+    ),
+    
+    'transgender rights usa' => test_spice(
```

Comment:
@moollaza Ahhh this is annoying! So I'm using `Locate::Country's` `country2code` function (on [line 33](https://github.com/duckduckgo/zeroclickinfo-spice/pull/1827/files#diff-97fe7be41867fb0ad4bbaf2c9095c707R33)) to check if the country name is valid,  I don't actually want the returned country alpha code. The issue is apparently in [version 2 of Locate::Country](http://sipi.usc.edu/manuals/perldoc-html/Locale/Country.html) some variant names were added meaning `country2code` will return a result for `us` or `usa`, which I don't really want as I'm checking for a valid full country name.  Any suggestions?